MultimodalQnA/tests/test_compose_on_xeon.sh

```diff
@@ -20,7 +20,9 @@ export caption_fn="apple.txt"
 
 function build_docker_images() {
     cd $WORKPATH/docker_image_build
-    git clone https://github.com/opea-project/GenAIComps.git && cd GenAIComps && git checkout "${opea_branch:-"main"}" && cd ../
+    # TODO: Revert to the clone of opea-project "main" after the merge of https://github.com/opea-project/GenAIComps/pull/852
+    # git clone https://github.com/opea-project/GenAIComps.git && cd GenAIComps && git checkout "${opea_branch:-"main"}" && cd ../
+    git clone --single-branch --branch="mmqna-phase2" https://github.com/mhbuehler/GenAIComps.git
```

Comment:
I think it's premature to have this here, since the pull # in the comment is not correct and the gaudi test file doesn't have this in it. Please revert for now, and we can do this step when upstreaming to OPEA.